pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -87,6 +91,10 @@ func NewConfigObserver(
 			apiserver.ObserveUserClientCABundle,
 			apiserver.ObserveAdditionalCORSAllowedOrigins,
 			auth.ObserveAuthMetadata,
+			encryption.NewEncryptionConfigObserver(
+				operatorclient.TargetNamespace,
+				[]string{"apiServerArguments", "encryption-provider-config"},
```

Comment:
why is this a parameter? Put it deep into the code where it is used.

Reply:
Or is it different for openshift-apiserver? (worth checking before following my suggestion :-) )

Reply:
will check :)

Reply:
It's the same for `openshift-apiserver` - it [takes](https://github.com/openshift/openshift-apiserver/blob/master/pkg/cmd/openshift-apiserver/openshiftapiserver/config.go#L38) [OpenShiftAPIServerConfig. APIServerArguments](https://github.com/openshift/api/blob/master/openshiftcontrolplane/v1/types.go#L45). No need to make it a param.